src/components/mobile-menu/mobile-menu.jsx

```diff
@@ -38,6 +39,17 @@ const MobileMenu = () => {
   return (
     <>
       <div className={classnames(styles.container, { [styles.openMenu]: showMenu })} >
+        {showTypeForm &&
+          <div className={styles.formContainer}>
+            <div onClick={() => setShowTypeForm(false)} className={styles.closeForm}><CloseIcon className={styles.closeFormIcon} /></div>
```

Comment:
this is to be a button. because it's clickable

Reply:
Done

Reply:
I don't see the change

Reply:
I'm sorry, I did it in another branch